src/blockfilter.h

```diff
@@ -168,4 +169,35 @@ class BlockFilter
     }
 };
 
+template <typename OStream>
+void GolombRiceEncode(BitStreamWriter<OStream>& bitwriter, uint8_t P, uint64_t x)
+{
+    // Write quotient as unary-encoded: q 1's followed by one 0.
+    uint64_t q = x >> P;
+    while (q > 0) {
+        int nbits = q <= 64 ? static_cast<int>(q) : 64;
+        bitwriter.Write(~0ULL, nbits);
```

Comment:
It might not be seeing the obvious (and I know this PR doesn't touches this), but what's the point of `~0ULL`? Seems confusing at a first glance.

 
```suggestion
        bitwriter.Write(0xFFFFFFFFFFFFFFFFULL, nbits);
```

Reply:
Yes, I agree that `~0` feels non-idiomatic. Personally I prefer `std::numeric_limits<uint64_t>::max()` :)

But as you note this PR does not touch that code -- it only moves it. Thus leaving the comment without action.